### ISSUE TITLE
I need to force reimport manually (delete all) feeds of Cielo on CSE

### ISSUE THREAD

**Issue by elbidone** [meumobi/sitebuilder#71]
I need to force reimport manually (delete all) feeds of Cielo on CSE
It means that import low priority didn't work

**Comment by elbidone** [comment]
Feed of category_id 94 (Triunfo pt) was out-dated friday 1st. It was updated on workr the 30th April. I ran a manual import the 1st afternoon and the importation works.

**Comment by elbidone** [comment]
I've reviewed all feeds of Cielo today. I've found that the last item of following feed, published this morning, was not imported on site-builder:
http://cielo.riweb.com.br/FeedMeuMobi.aspx?idCanal=Wcd4QWcnf4RWIXLQm2cXkQ==&tp=0

I've checked on mongo, the last item (higher order, OBS: why order use a range of [10-59], instead of [0-49]?) was published the 14th April.
```
> db.items.find({parent_id: 212}).sort({order: -1}).limit(5)
{ "_id" : ObjectId("553f67be70179604458b45a2"), "order" : 59, "created" : ISODate("2015-04-14T19:51:00Z"), "modified" : ISODate("2015-04-28T10:58:06.355Z"), "published" : ISODate("2015-04-28T10:58:06.354Z"), "is_published" : true, "thumbnails" : [ ], "groups" : [ ], "guid" : "http://cielo.riweb.com.br/ShowApresentacao.aspx?IdTeleconferencia=WYSSayp3TGJuJYmyS3fz2A==", "link" : "http://cielo.riweb.com.br/ShowApresentacao.aspx?IdTeleconferencia=WYSSayp3TGJuJYmyS3fz2A==", "pubdate" : ISODate("2015-04-14T19:51:00Z"), "description" : "", "author" : "", "medias" : [       {       "url" : "http://cielo.riweb.com.br/Download.aspx?Arquivo=ojlKC7IpQ87xI4neyiaCPw==",         "type" : "application/pdf",     "title" : "20150413--Cielo_Governance_port_v2",    "length" : 1417,         "thumbnails" : [ ] } ], "site_id" : 15, "parent_id" : 212, "title" : "Apresentação Governança Corporativa", "format" : "html", "type" : "articles" }
{ "_id" : ObjectId("553f67be70179604458b45a1"), "order" : 58, "created" : ISODate("2015-04-06T13:00:00Z"), "modified" : ISODate("2015-04-28T10:58:06.292Z"), "published" : ISODate("2015-04-28T10:58:06.291Z"), "is_published" : true, "thumbnails" : [ ], "groups" : [ ], "guid" : "http://cielo.riweb.com.br/ShowApresentacao.aspx?IdTeleconferencia=+ldCtTVNn0GKjyIj9q74uw==", "link" : "http://cielo.riweb.com.br/ShowApresentacao.aspx?IdTeleconferencia=+ldCtTVNn0GKjyIj9q74uw==", "pubdate" : ISODate("2015-04-06T13:00:00Z"), "description" : "", "author" : "", "medias" : [       {       "url" : "http://cielo.riweb.com.br/Download.aspx?Arquivo=q1nz77JF/aTsB9XcIPXOiw==",         "type" : "application/pdf",     "title" : "20150401_Cielo_Abril_v3",    "length" : 2030,    "thumbnails" : [ ] } ], "site_id" : 15, "parent_id" : 212, "title" : "Apresentação Institucional 2015", "format" : "html", "type" : "articles" }
{ "_id" : ObjectId("553f67be70179604458b45a0"), "order" : 57, "created" : ISODate("2015-03-31T16:40:00Z"), "modified" : ISODate("2015-04-28T10:58:06.229Z"), "published" : ISODate("2015-04-28T10:58:06.228Z"), "is_published" : true, "thumbnails" : [ ], "groups" : [ ], "guid" : "http://cielo.riweb.com.br/ShowApresentacao.aspx?IdTeleconferencia=RbLZdIiXWk4Hs5YJMNZlTQ==", "link" : "http://cielo.riweb.com.br/ShowApresentacao.aspx?IdTeleconferencia=RbLZdIiXWk4Hs5YJMNZlTQ==", "pubdate" : ISODate("2015-03-31T16:40:00Z"), "description" : "", "author" : "", "medias" : [       {       "url" : "http://cielo.riweb.com.br/Download.aspx?Arquivo=hCLhx Ce/3yDWFr3rq0W5Q==",         "type" : "application/pdf",     "title" : "20150327_APIMEC com selo APIMEC",    "length" : 1392,    "thumbnails" : [ ] } ], "site_id" : 15, "parent_id" : 212, "title" : "Apresentação Cielo APIMEC/SP 2015", "format" : "html", "type" : "articles" }
```

I've tried to find stg on logs without success
```
# cat log/php.log |grep 1T15
#
```

**Comment by elbidone** [comment]
I've reimported the same feed but on a new hidden category

Here the result:
```
> db.items.find({parent_id: 457}).sort({order: -1}).limit(3)
{ "_id" : ObjectId("5547d5dc701796e1378b45a3"), "order" : 60, "created" : ISODate("2015-04-29T13:05:00Z"), "modified" : ISODate("2015-05-04T20:26:04.720Z"), "published" : ISODate("2015-05-04T20:26:04.718Z"), "is_published" : true, "thumbnails" : [ ], "groups" : [ ], "guid" : "http://cielo.riweb.com.br/Show.aspx?idMateria=zPGTGjQAQXvNAkopIk5zBA==", "link" : "http://cielo.riweb.com.br/Show.aspx?idMateria=zPGTGjQAQXvNAkopIk5zBA==", "pubdate" : ISODate("2015-04-29T13:05:00Z"), "description" : "", "author" : "", "medias" : [   {       "url" : "http://cielo.riweb.com.br/Download.aspx?Arquivo=6D3zrKQuA6GWz0Ic7RTU5A==",         "type" : "application/pdf",         "title" : "20150428_Webcast_1T15_VF",   "length" : 769,         "thumbnails" : [ ] } ], "site_id" : 15, "parent_id" : 457, "title" : "Apresentação Resultados 1T15", "format" : "html", "type" : "articles" }
{ "_id" : ObjectId("5547d5dc701796e1378b45a2"), "order" : 59, "created" : ISODate("2015-04-14T19:51:00Z"), "modified" : ISODate("2015-05-04T20:26:04.679Z"), "published" : ISODate("2015-05-04T20:26:04.678Z"), "is_published" : true, "thumbnails" : [ ], "groups" : [ ], "guid" : "http://cielo.riweb.com.br/ShowApresentacao.aspx?IdTeleconferencia=WYSSayp3TGJuJYmyS3fz2A==", "link" : "http://cielo.riweb.com.br/ShowApresentacao.aspx?IdTeleconferencia=WYSSayp3TGJuJYmyS3fz2A==", "pubdate" : ISODate("2015-04-14T19:51:00Z"), "description" : "", "author" : "", "medias" : [       {       "url" : "http://cielo.riweb.com.br/Download.aspx?Arquivo=ojlKC7IpQ87xI4neyiaCPw==",         "type" : "application/pdf",     "title" : "20150413--Cielo_Governance_port_v2",    "length" : 1417,         "thumbnails" : [ ] } ], "site_id" : 15, "parent_id" : 457, "title" : "Apresentação Governança Corporativa", "format" : "html", "type" : "articles" }
{ "_id" : ObjectId("5547d5dc701796e1378b45a1"), "order" : 58, "created" : ISODate("2015-04-06T13:00:00Z"), "modified" : ISODate("2015-05-04T20:26:04.638Z"), "published" : ISODate("2015-05-04T20:26:04.636Z"), "is_published" : true, "thumbnails" : [ ], "groups" : [ ], "guid" : "http://cielo.riweb.com.br/ShowApresentacao.aspx?IdTeleconferencia=+ldCtTVNn0GKjyIj9q74uw==", "link" : "http://cielo.riweb.com.br/ShowApresentacao.aspx?IdTeleconferencia=+ldCtTVNn0GKjyIj9q74uw==", "pubdate" : ISODate("2015-04-06T13:00:00Z"), "description" : "", "author" : "", "medias" : [       {       "url" : "http://cielo.riweb.com.br/Download.aspx?Arquivo=q1nz77JF/aTsB9XcIPXOiw==",         "type" : "application/pdf",     "title" : "20150401_Cielo_Abril_v3",    "length" : 2030,    "thumbnails" : [ ] } ], "site_id" : 15, "parent_id" : 457, "title" : "Apresentação Institucional 2015", "format" : "html", "type" : "articles" }
```

Some observations:

- I don't understand the meaning of field created_date, because it's always the pubdate and not the date of creation on db as expected
- the feed has no problem, that means that our importation is responsible for the bug

On class UpdateFeedService I've added logs:

```
$feed_stats = $extension->updateArticles();
        $stats['total_articles'] += $feed_stats['total_articles'];
        $stats['removed_articles'] += $feed_stats['removed_articles'];
        $stats['total_images'] += $feed_stats['total_images'];
        $stats['failed_images'] += $feed_stats['failed_images'];
        $stats['total_feeds'] += 1;

        $this->logger()->info('End of Extension Processing', [
          'Extension Id' => $extensionData['_id'],
          'Category Id' => $extensionData['category_id'],
          'Total Articles' => $feed_stats['total_articles'],
          'Removed Articles' => $feed_stats['removed_articles']
          ]);

        $connection->extensions->update([
```

And I've seen that the extension "_id" : ObjectId("55403570701796d23d8b4567") linked to category 212 is never processed.

```
> db.extensions.find({category_id: 212})
{ "_id" : ObjectId("55403570701796d23d8b4567"), "category_id" : 212, "created" : ISODate("2015-04-29T01:35:44.689Z"), "enabled" : 1, "extension" : "rss", "modified" : ISODate("2015-04-29T01:35:44.690Z"), "priority" : 0, "site_id" : 15, "url" : "http://cielo.riweb.com.br/FeedMeuMobi.aspx?idCanal=Wcd4QWcnf4RWIXLQm2cXkQ==&tp=0", "use_html_purifier" : 1 }
```

**Comment by elbidone** [comment]
Due to #70 low priority was not processed because the implementation didn't consider it.
